python/lsst/validate/drp/matchreduce.py

```diff
@@ -295,73 +314,95 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
     return srcVis, allMatches
 
 
-def _reduceStars(blob, allMatches, safeSnr=50.0):
+def _reduceSources(blob, allMatches, goodSnr=3.0, safeSnr=50.0, safeExtendedness=1.0, extended=False,
+                   nameFluxKey=None, goodSnrMax=np.Inf, safeSnrMax=np.Inf):
     """Calculate summary statistics for each star. These are persisted
     as object attributes.
 
     Parameters
     ----------
     allMatches : `lsst.afw.table.GroupView`
         GroupView object with matches.
-    safeSnr : `float`, optional
-        Minimum median SNR for a match to be considered "safe".
+    goodSnr : float, optional
+        Minimum median SNR for a match to be considered "good"; default 3.
```

Comment:
I think goodSnr should default to at least 5.  There are a lot of 3-sigma detections in ~billion pixels.